Makefile

```diff
@@ -350,10 +350,15 @@ RELEASE_DIR_K8S_MANIFESTS?=$(RELEASE_DIR)/k8s-manifests
 RELEASE_DIR_IMAGES?=$(RELEASE_DIR)/images
 RELEASE_DIR_BIN?=$(RELEASE_DIR)/bin
 
-# Determine where the manifests live. For older versions we used
-# a different location, but we still need to package them up for patch
-# releases.
+# Determine where the manifests live.
+ifneq ("$(wildcard $(RELEASE_STREAM))","")
```

Comment:
can we just delete this conditional altogether now that manifests always exist in `_site/manifests`?

Reply:
Yeah good point. PTAL @ozdanborne